### PR TITLE
LowKVMNodesCount edits

### PR DESCRIPTION
**Pull request by apinnick**
Signed-off-by: Avital Pinnick <apinnick@redhat.com>

### CODE REVIEW

File: docs/runbooks/LowKVMNodesCount.md [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
 ## Diagnosis
 
-Verify that nodes have KVM resource available.
+- Identify the nodes with KVM resources:
 ```
-kubectl get nodes -o jsonpath='{.items[*].status.allocatable}' | grep devices.kubevirt.io/kvm
+$ kubectl get nodes -o jsonpath='{.items[*].status.allocatable}' | grep devices.kubevirt.io/kvm
 ```
 
 ## Mitigation
```
Comment (apinnick) [inline docs/runbooks/LowKVMNodesCount.md:22]: @andreyod @sradco 
Are there any mitigations for downstream?

**Comment by apinnick** [comment]
@sradco Ready for review. It would be nice if we could provide D/S mitigation.

**Comment by sradco** [comment]
@enp0s3 Hi, can you please see if there is a d/s mitigation that we can propose?

**Comment by enp0s3** [comment]
@sradco Hi. AFAIK the only way to mitigate is to install KVM on nodes where its missing. Other options would be workarounds (i.e. use emulator mode, which is slow and it won't consider the nodes where KVM present)

**Comment by apinnick** [comment]
> @sradco Hi. AFAIK the only way to mitigate is to install KVM on nodes where its missing. Other options would be workarounds (i.e. use emulator mode, which is slow and it won't consider the nodes where KVM present)

That sounds reasonable. I'll add it.

**Comment by apinnick** [comment]
@enp0s3 @sradco 
For installing KVM, do you think a link to [RHEL docs](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html-single/configuring_and_managing_virtualization/index#assembly_enabling-virtualization-in-rhel-9_configuring-and-managing-virtualization) would be OK?

**Comment by apinnick** [comment]
> @apinnick Its a bit complicated, because cluster node OS can be RHCOS as well. In case of RHCOS, deployment of the KVM module must be done by one of cluster node operators, such as the [SRO](https://access.redhat.com/documentation/si-lk/openshift_container_platform/4.9/html/specialized_hardware_and_driver_enablement/special-resource-operator), I don't know the accurate answer here, it may be that more than one operator needs to be involved, such as Node Tuning Operator, maybe the Machine Config Operator as well.

@enp0s3 I understand. Since runbooks are not meant to be substitutes for the documentation, I will just use the simplest wording with no link. Thanks.

**Pull request merged**